controller/tenant_updates.go

```diff
@@ -0,0 +1,111 @@
+package controller
+
+import (
+	"github.com/fabric8-services/admin-console/app"
+	"github.com/fabric8-services/admin-console/application"
+	"github.com/fabric8-services/admin-console/auditlog"
+	"github.com/fabric8-services/fabric8-common/errors"
+	"github.com/fabric8-services/fabric8-common/httpsupport"
+	"github.com/fabric8-services/fabric8-common/token"
+	"github.com/goadesign/goa"
+)
+
+// TenantUpdatesController implements the TenantUpdates resource.
+type TenantUpdatesController struct {
+	*goa.Controller
+	config TenantUpdatesControllerConfiguration
+	db     application.DB
+}
+
+// TenantUpdatesControllerConfiguration the configuration for the SearchController
+type TenantUpdatesControllerConfiguration interface {
+	GetTenantServiceURL() string
+}
+
+// NewTenantUpdatesController creates a TenantUpdates controller.
+func NewTenantUpdatesController(service *goa.Service, config TenantUpdatesControllerConfiguration, db application.DB) *TenantUpdatesController {
+	return &TenantUpdatesController{
+		Controller: service.NewController("TenantUpdatesController"),
+		config:     config,
+		db:         db,
+	}
+}
+
+// Show returns information about the ongoing tenant update
+func (c *TenantUpdatesController) Show(ctx *app.ShowTenantUpdatesContext) error {
+	tokenManager, err := token.ReadManagerFromContext(ctx)
+	if err != nil {
+		return app.JSONErrorResponse(ctx, errors.NewUnauthorizedError("missing token manager in the request context"))
+	}
+	identityID, err := tokenManager.Locate(ctx)
+	if err != nil {
+		return app.JSONErrorResponse(ctx, errors.NewUnauthorizedError("invalid authorization token (invalid 'sub' claim)"))
+	}
+	record := auditlog.AuditLog{
+		EventTypeID: auditlog.ShowTenantUpdate,
+		IdentityID:  identityID,
+		EventParams: auditlog.EventParams{},
+	}
+	err = application.Transactional(c.db, func(appl application.Application) error {
+		return appl.AuditLogs().Create(ctx, &record)
+	})
+	if err != nil {
+		return app.JSONErrorResponse(ctx, err)
+	}
+	return httpsupport.RouteHTTP(ctx, c.config.GetTenantServiceURL())
```

Comment:
let me clarify one thing - this endpoint is using `/tenant/updates` path and the `c.config.GetTenantServiceURL()` returns just `f8tenant` - shouldn't we specify also the target path in the tenant service here: `/api/update` ?

Reply:
didn't noticed that you have already realized it :-) https://github.com/fabric8-services/admin-console/pull/48#issuecomment-454300791

Reply:
yes, that's what I need to change as well (see https://github.com/fabric8-services/admin-console/pull/48#issuecomment-454300791)

Reply:
done in 02cc202